user.go

```diff
@@ -122,6 +128,11 @@ func (u *UserService) ListByTag(tagID string, params PageParams) (UserList, erro
 	return u.Repository.list(userListParams{PageParams: params, TagID: tagID})
 }
 
+// Scroll lists all Users for App.
+func (u *UserService) Scroll(scrollParam string) (UserScroll, error) {
+	return u.Repository.scroll(scrollParam)
```

Comment:
I think it's better to wrap this value to `ScrollParams` here and then pass it to `UserRepository`. It's more consistent with rest of `UserService`